lib/matplotlib/colors.py

```diff
@@ -87,6 +87,7 @@ def __delitem__(self, key):
 _colors_full_map = _ColorMapping(_colors_full_map)
 
 _REPR_PNG_SIZE = (512, 64)
+_BIVAR_REPR_PNG_SIZE = 256
```

Comment:
I thought they would be a bit too big if they are 512×512, so I set them to 256 (which is also the default size of the look up table). Since they are rendered as a square, they only need one number, but we could always change this to two numbers.

Here is an example of the size:
![image](https://github.com/user-attachments/assets/98fe9879-c70f-489d-a1f0-3c825446febe)